src/components/animated/_colored-on-hover.less

```diff
@@ -0,0 +1,108 @@
+@keyframes fonnt {
+	0% {
+		color: black;
+	}
+	100% {
+		color: cornsilk;
+		text-shadow: 0px 0px 20px white;
+		box-shadow: inset 0px 0px 5px cornsilk;
+	}
+}
+@keyframes fonntrev {
+	0% {
+		color: black;
+	}
+	100% {
+		color: black;
+		box-shadow: inset 0px 0px 5px black;
+	}
+}
+.sbtn.colored-on-hover-btn {
+	position: relative;
+	padding: 0px;
+	color: black;
+	height: 50px;
+	width: 200px;
+	background-color: transparent;
+	cursor: pointer;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	outline: none;
+	&:hover {
+		animation: fonnt 0.25s ease-in forwards;
+	}
+	.bluebtn{
```

Comment:
add `&` before all the button colors. Also, you have this button color named wrong. It should be `blue-btn`